packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -354,18 +353,50 @@ export default function TypographyPanel( {
 
 	// Text Orientation
 	const hasTextOrientationControl = useHasTextOrientationControl( settings );
-	const textOrientation = decodeValue(
-		inheritedValue?.typography?.textOrientation
-	);
-	const setTextOrientation = useCallback(
+
+	const setWritingModeAndTextOrientation = useCallback(
 		( newValue ) => {
-			onChange(
-				setImmutably(
-					value,
-					[ 'typography', 'textOrientation' ],
-					newValue || undefined
-				)
+			let newWritingMode, newTextOrientation;
+			switch ( newValue ) {
+				case 'bottom-to-top':
+					newTextOrientation = undefined;
+					newWritingMode = 'sideways-lr';
```

Comment:
Sideways is not supported by browsers:
https://caniuse.com/?search=sideways
https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode#browser_compatibility

Reply:
Oh good to know thanks. We can just use vertical and mixed then :)